options/options_test.go

```diff
@@ -25,6 +27,54 @@ const (
 	ShouldFail
 )
 
+func TestLogUnsupportedOptions(t *testing.T) {
+	testtype.SkipUnlessTestType(t, testtype.IntegrationTestType)
+
+        Convey("With all command-line options enabled", t, func() {
+		// Setup code is duplicated for both Convey-blocks to
```

Comment:
You don't need to do this. You should look at the suggestion I made and read about the [convey execution order](https://github.com/smartystreets/goconvey/wiki/Execution-order). The outer convey is run separately for each inner convey.

Reply:
Ahh, I understand now. Thanks for explaining it, once again!